packages/crayons-core/src/components/select-option/select-option.tsx

```diff
@@ -205,6 +205,24 @@ export class SelectOption {
   }
 
   createDescription() {
+    if ((this.subText && this.metaText) || this.metaText) {
```

Comment:
@harshith-venkatesh-freshworks Can we try to unify the method createDescription and createConversationDescription? There are some duplicated codes in these methods which can be further optimised. Also I think we won't be needing the variant as conversation if we allow metaText to be accepted without the variant. We can use variant as icon to display the icon as well as metaText. Can you please check them?

Reply:
@rihansiddhi , good point, Indeed we can remove createConversationDescription here.
However, for the icon.
Are we able to customise the padding for the display.
I have made some changes to conversation variant.

Reply:
Let me know your view on the same

Reply:
```suggestion
    if (this.metaText) {
```
@harshith-venkatesh-freshworks Can we simplify this as we are already checking subText condition inside so adding a condition outside wouldn't make any difference right?